Utils/User.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Assignment_1.Enums;
+using System.Collections.Generic;
+
+namespace Assignment_1.Utils
+{
+    /// <summary>
+    /// Represents a user in the library system.
+    /// </summary>
+    internal class User
+    {
+        /// <summary>
+        /// Gets the unique identifier of the user.
+        /// </summary>
+        public int UserId { get; }
+
+        /// <summary>
+        /// Gets the name of the user.
+        /// </summary>
+        public string Name { get; }
+
+        /// <summary>
+        /// Gets the type of the user (Student or Teacher).
+        /// </summary>
+        public UserType Type { get; }
+
+        /// <summary>
+        /// Gets the list of books borrowed by the user.
+        /// </summary>
+        public List<Book> BorrowedBooks { get; } = new List<Book>();
+
+        /// <summary>
+        /// Gets or sets the fine amount imposed on the user.
+        /// </summary>
+        public double Fine { get; private set; }
```

Comment:
I know adding doc is a good practice but adding doc on every line is not some wants. You should only add the doc to relevant places. For eg. Above a complex code to explain the logic and edge cases or above a function definition.